src/app/calculator/furnaces/opening/opening.service.ts

```diff
@@ -208,6 +208,10 @@ export class OpeningService {
     };
   }
 
+    viewFactorCalculation(input: any, settings: Settings): number {
```

Comment:
Wrapping this so I don't need to bring phastService into the form ts

Reply:
put a type on the inputs